src/System.Linq.Parallel/tests/QueryOperators/AggregateTests.cs

```diff
@@ -304,6 +304,17 @@ public static void Aggregate_OperationCanceledException(Labeled<ParallelQuery<in
         }
 
         [Theory]
+        [MemberData(nameof(UnorderedSources.Ranges), new[] { 128 }, MemberType = typeof(UnorderedSources))]
```

Comment:
Do we need to do these for all of these different sources? (Same question for all of these other tests.)  The PLINQ tests are in general doing more of a full matrix than I'd like, resulting in a lot of duplication.  In some cases, where the test is verifying functionality that is impacted by the sources, this makes perfect sense, but that's not the case here.  These tests really only depend on correctly behaving operators.